inspectors/models/abstractinspectormodel.cpp

```diff
@@ -277,7 +281,9 @@ void AbstractInspectorModel::loadPropertyItem(PropertyItem* propertyItem, std::f
     bool isUndefined = false;
 
     for (const Ms::Element* element : m_elementList) {
-
+        if (!element)
+            continue;
```

Comment:
Is this a typical or exceptional situation? If exceptional, it is best to use
```
IF_ASSERT_FAILED(score) {
    return;   
}
```

If typical, it is better to write a comment about why there may not be score